### DISCUSSION TITLE
Zero (and one?) adjusted Dirichlet?

### DISCUSSION THREAD

**Discussion by p-schaefer** [StatMixedML/XGBoostLSS#41] [Ideas]
Would it be possible to implement  a 0 (and maybe even 0 and 1) adjusted dirichlet distribution, similar to: 

Tsagris, M., & Stewart, C. (2018). A Dirichlet Regression Model for Compositional Data with Zeros. Lobachevskii Journal of Mathematics, 39(3), 398–412. doi:10.1134/s1995080218030198

**Comment by StatMixedML** [comment]
Thanks for your interest in the project.

This is similar to https://github.com/StatMixedML/LightGBMLSS/issues/17. I haven't really considered extending the Dirichlet in that way, but I add it to the ToDos. Thanks for pointing me to this!
